src/csvcubed/cli/inspect/metadataprinter.py

```diff
@@ -195,13 +200,29 @@ def generate_codelist_results(self):
         self.result_code_list_cols = select_codelist_cols_by_dataset_url(
             self.csvw_metadata_rdf_graph, self.dataset_url
         )
+        # Retrieving the primary key of the code list to identify the unique identifier
+        result_primary_keys_by_dataset_url: PrimaryKeysByDatasetUrlResult = (
```

Comment:
I'd call it primary key columns instead of keys, keys implies there are multiple different indexes/unique values.